### PR TITLE
Fix syntax error preventing rollout

### PR DESCRIPTION
**Pull request by mjlshen**
### What type of PR is this?
bug

### What this PR does / why we need it?
The rollout or #2077 is currently failing due to
```
[2024-04-11 19:49:12] [ERROR] [openshift_base.py:apply_action:864] - [hives03ue1/cluster-scope] [https://api.hives03ue1.2yi8.p1.openshiftapps.com:6443/]: The SelectorSyncSet "osd-ingress-routerreplicas-osd-20989" is invalid: 
* spec.clusterDeploymentSelector.matchExpressions[0].values: Invalid value: "string": spec.clusterDeploymentSelector.matchExpressions[0].values in body must be of type array: "string"
* spec.clusterDeploymentSelector.matchExpressions[1].values: Invalid value: "string": spec.clusterDeploymentSelector.matchExpressions[1].values in body must be of type array: "string"
```

### Which Jira/Github issue(s) this PR fixes?
[OSD-20989](https://issues.redhat.com//browse/OSD-20989)

### CODE REVIEW

**Comment by mjlshen** [comment]
/hold

This will need a prerequisite change in app-interface before this should be merged. Anyone feel free to unhold when that's the case

**Review by fahlmant — APPROVED** [review]
/approve
/lgtm

**Pull request merged**
